### PR TITLE
Publish to NuGet

### LINKED ISSUE #4: NuGet package
Hi, first of all thanks for building this fancy task builder! From my first few tests it works like a charm! I came across it, because someone used your TaskBuilder.fs to improve the performance of the [Giraffe](https://github.com/dustinmoris/Giraffe) web framework and therefore I would be interested in consuming it via a NuGet package rather than having to copy/paste it every time you improve it or bug fix something. It would also make it easier by having it as a well documented dependency and avoid other contributors to try and change the TaskBuilder.fs in Giraffe when really they should probably discuss any changes/improvements with you here first so that everyone can benefit from it.

Would you be interested in making your TaskBuilder.fs accessible via a NuGet library? If yes I can even help out with a PR myself. Let me know and thank you for open sourcing it!

### PR DESCRIPTION
**Pull request by ForNeVeR**
Hello! I've decided to add the package for .NET Standard 1.6 (because I currently need only 1.6+).

For now, I haven't registered https://www.nuget.org/packages/TaskBuilder.fs/, although I have no problems registering it and maintaining NuGet package if @rspeele is okay with it (I will also share NuGet ownership with anyone @rspeele wants).

Closes #4.

/cc @dustinmoris (you were interested in the packaging)

Currently it's [published on MyGet](https://www.myget.org/feed/fornever/package/nuget/fornever.TaskBuilder.fs) if anybody wants to try it.

### CODE REVIEW

File: TaskBuilder.fs.fsproj [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <PackageVersion>1.0.0</PackageVersion>
+    <PackageId>TaskBuilder.fs</PackageId>
+    <Title>TaskBuilder.fs</Title>
+    <Authors>ForNeVeR</Authors>
```
Comment (ForNeVeR) [inline TaskBuilder.fs.fsproj]: Please note that I'm not claiming authorship of the code, but NuGet spec [tells me](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#nuget-metadata-properties) to use "list of packages authors, matching the profile names on nuget.org." I couldn't found your NuGet account, @rspeele. If you haven't registered there, then we probably could just leave this as-is.

**Comment by rspeele** [comment]
Hey,

Thanks so much for contributing this! I have not messed with any .NET Core/.NET Standard stuff yet (still working on boring old .NET Framework projects) so it's all new to me.

I do have a NuGet account here: https://www.nuget.org/profiles/humbobst

If you add that to the authors list I'll happily merge this!

**Comment by ForNeVeR** [comment]
Done. If you want to publish the package yourself, I could also remove myself from the author list — or feel free to merge the PR and remove me afterwards.

_(I am okay with managing the package and I'll share the ownership with you if that's more convenient — you choose.)_

**Comment by rspeele** [comment]
Excellent -- feel free to publish it yourself and keep yourself on the authors list! Thank you!

**Comment by ForNeVeR** [comment]
Alright, done. You're now a co-owner of the package https://www.nuget.org/packages/TaskBuilder.fs/. Readme badge will be updated soon automatically, thanks to the shields.io magic.

Also, regarding the "boring old .NET Framework projects": I am working on them myself on my work, that's totally okay :)

Theoretically (any by that I mean that I haven't tried that personally, but Microsoft promises it should work), .NET projects should be able to consume .NET Standard packages.

We're using .NET Standard 1.6 in our package (that's the lowest we could use with F#), and according to [the monstrous compatibility matrix](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) that means that .NET 4.6.1 projects "(with .NET Core 2.0 SDK)" (whatever that means) should be able to consume the package.

In case anyone wants us to support other versions of .NET — I'm okay with that, we could change our `fsproj` to generate those. That'll be a bit harder (because I've never done that), bu I'll do my best to help F# guys over all the frameworks and platforms.

**Comment by quezlatch** [comment]
hi. i've added a pr https://github.com/rspeele/TaskBuilder.fs/pull/16 to support older frameworks.

**Pull request merged**